src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/RemoveOrganizationUserCommand.cs

```diff
@@ -106,21 +114,33 @@ public async Task<List<Tuple<OrganizationUser, string>>> RemoveUsersAsync(Guid o
                     throw new BadRequestException("Only owners can delete other owners.");
                 }
 
+                if (managementStatus.TryGetValue(orgUser.Id, out var isManaged) && isManaged)
+                {
+                    throw new BadRequestException("Managed members cannot be simply removed, their entire individual account must be deleted.");
+                }
```

Comment:
This command is also used by SCIM which can continue to remove managed users (integrations are excluded from the delete behavior until we figure out how to do it safely). So there needs to be some conditionality here. Maybe stealing `actingUser` from Jared's PR would be useful so it can encapsulate `deletingUserId` or the `SystemUser` (and in that case you can exempt SCIM from this check).

I don't love it but I think my problem is with the diverging business logic. Handling it in the command like this seems reasonable.

Reply:
I have refactored this so that we don't check managed users when an `EventSystemUser` is passed.